src/translate.py

```diff
@@ -1320,6 +1320,7 @@ def format(self, fmt: Formatter) -> str:
 class Literal(Expression):
     value: int
     type: Type = field(compare=False, default_factory=Type.any)
+    elide_cast: bool = False
```

Comment:
```suggestion
    elide_cast: bool = field(compare=False, default=False)
```

This way `Literal(1, elide_cast=True) == Literal(1, elide_cast=False)`, [ref](https://docs.python.org/3/library/dataclasses.html).